src/FiltrationGroupComponent/FiltrationGroupComponent.tsx

```diff
@@ -121,23 +130,29 @@ export class FiltrationGroupComponent extends React.Component<Props> {
 	};
 
 	public handleCreateVerticalBrackets = () => {
+		const { verticalBracket } = this.state;
+
+		let isVerticalBracket = false;
+
 		if (this.kitFiltrationRef && this.kitFiltrationRef.current) {
 			const ref = this.kitFiltrationRef.current;
 			const parentNode = ref.parentNode as HTMLElement;
 
 			if (parentNode.classList.contains("kit-filtration-condition")) {
-				const label = this.kitFiltrationLabelRef.current;
-				const verticalBracket = document.createElement("span");
-
-				verticalBracket.className =
-					"kit-filtration-group__label-vertical-bracket";
-
-				label && label.appendChild(verticalBracket);
+				isVerticalBracket = true;
 			}
 		}
+
+		if (isVerticalBracket !== verticalBracket) {
+			this.setState({
+				verticalBracket: !verticalBracket
+			});
+		}
 	};
 
 	public handleCreateHorizontalBrackets = () => {
+		const { horizontalBracket } = this.state;
+		let horizontalBrackets = [] as any;
```

Comment:
`any` опасная штука, лучше не использовать.
Тут у тебя
```suggestion
		let horizontalBrackets: PositionTop[] = [];
```
или если мы уберём объект
```suggestion
		let horizontalBrackets; number[] = [];
```